mock/py/mockbuild/plugins/root_cache.py

```diff
@@ -7,11 +7,13 @@
 # python library imports
 import fcntl
 import os
+import subprocess
 import time
 
 # our imports
 from mockbuild.trace_decorator import getLog, traceLog
 import mockbuild.util
+from mockbuild.util import do
```

Comment:
Can you instead of `do()` call `mockbuild.util.do()`? It is just nitpicking, but it is used this way everywhere in the code.

Reply:
No, that's perfectly fine. I've changed that.